ets_fiber_assigner/netflow.py

```diff
@@ -564,6 +578,13 @@ def buildProblem(bench, targets, tpos, classdict, tvisit, vis_cost=None,
                 prob.add_constraint(makeName("InstReg",ivis,i),
                     prob.sum([v for v in regionVars[ivis][i]]) >= minSkyTargetsPerInstrumentRegion)
 
+    # Make sure that enough fibers are kept unassigned, if this was requested
+    if numReservedFibers > 0:
+        maxAssignableFibers = bench.cobras.nCobras-numReservedFibers
+        for ivis in range(nvisits):
+            relevantVars = [var for (keys, var) in Cv_i.items() if keys[1] == ivis]
```

Comment:
Again, the following modification works for me.
```suggestion
            relevantVars = [var[0] for (keys, var) in Cv_i.items() if keys[1] == ivis]
```